cpp/velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -479,13 +494,14 @@ class SubstraitToVeloxPlanConverter {
   template <TypeKind KIND, typename FilterType>
   void createNotEqualFilter(variant notVariant, bool nullAllowed, std::vector<std::unique_ptr<FilterType>>& colFilters);
 
-  /// Create a values range to handle in filter.
-  /// variants: the list of values extracted from the in expression.
+  /// Create a values range to handle (not) in filter.
+  /// variants: the list of values extracted from the (not) in expression.
```

Comment:
Is the behavior to create `in` or `not in` controlled by the variable `negated`?

Reply:
yes, add comments for this param